microservices-dashboard-server/pom.xml

```diff
@@ -39,6 +39,11 @@
             <name>Tim Ysewyn</name>
             <email>tim.ysewyn@ordina.be</email>
         </developer>
+        <developer>
```

Comment:
When updating the project, please also update the version of the `spring-boot-starter-parent` to the latest version.